frontend/src/metabase/visualizations/lib/apply_tooltips.unit.spec.js

```diff
@@ -211,27 +211,20 @@ describe("getStackedTooltipModel", () => {
     ordered_cards: [],
   };
   const cols = [StringColumn(), NumberColumn()];
+  const getMockSerie = ({ card, rows, hasBreakout }) => {
+    const _breakoutColumn = hasBreakout ? StringColumn() : undefined;
+    return {
+      data: { cols, rows, _breakoutColumn },
+      card: { ...card, _breakoutColumn },
+    };
+  };
   const getMockSeries = hasBreakout => [
-    {
-      data: {
-        cols,
-        rows: [["foo", 100]],
-        settings: {},
-        _breakoutColumn: hasBreakout ? StringColumn() : undefined,
-      },
-      card: {
-        name: "Series 1",
-        _breakoutColumn: hasBreakout ? StringColumn() : undefined,
-      },
-    },
-    {
-      data: {
-        cols,
-        rows: [["foo", 200]],
-        settings: {},
-      },
-      card: { name: "Series 2" },
-    },
+    getMockSerie({
```

Comment:
nit: since `serie` is an obsolete word we in other places of the codebase we use `series` and `multipleSeries`

Reply:
✅ 4602b8c547227736d2ec1d05e24b79d8bb38aa7f